Mlem/Views/Shared/Links/User Profile Label.swift

```diff
@@ -27,55 +27,101 @@ struct UserProfileLabel: View {
+                    } else {
+                        Image(systemName: "person.circle")
+                            .resizable()
+                            .scaledToFill()
+                            .frame(width: AppConstants.largeAvatarSize, height: AppConstants.defaultAvatarSize)
+                            .foregroundColor(.secondary)
+                    }
+                }
+            } else {
+                Image(systemName: "person.circle")
+                    .resizable()
+                    .scaledToFill()
+                    .frame(width: AppConstants.largeAvatarSize, height: AppConstants.defaultAvatarSize)
+                    .foregroundColor(.secondary)
             }
+        }
+        .frame(width: AppConstants.largeAvatarSize, height: AppConstants.largeAvatarSize)
+        .clipShape(Circle())
+        .overlay(Circle()
+            .stroke(Color(UIColor.secondarySystemBackground), lineWidth: 1))
+        .accessibilityHidden(true)
+    }
 
-            HStack(spacing: 0) {
-                // User display name if one exists
+    @ViewBuilder
+    private var userName: some View {
+        let flair = calculateLinkFlair()
+        
+        VStack(alignment: .leading) {
+            HStack(spacing: 4) {
+                if let flairImage = flair.image {
+                    flairImage
+                        .resizable()
+                        .scaledToFill()
+                        .frame(width: 12, height: 12) // TODO: scale with font
+                        .clipShape(RoundedRectangle(cornerRadius: 2))
+                }
+                
                 Text(user.displayName ?? user.name)
+                    .font(.footnote)
+                    .bold()
+            }
+            .foregroundColor(flair.color)
+            if showServerInstance, let host = user.actorId.host() {
+                Text("@\(host)")
                     .minimumScaleFactor(0.01)
                     .lineLimit(1)
-                    .bold()
-                    .foregroundColor(flair.color)
-                
-                if showServerInstance {
-                    if let host = user.actorId.host() {
-                        Text("@\(host)")
-                            .minimumScaleFactor(0.01)
-                            .lineLimit(1)
-                            .foregroundColor(flair.color)
-                            .opacity(0.6)
-                    }
-                }
+                    .opacity(0.6)
+                    .font(.caption)
             }
         }
+        .foregroundColor(.secondary)
     }
-
+    
     struct UserProfileLinkFlair {
         var color: Color
-        var systemIcon: String?
+        var image: Image?
     }
 
     private func calculateLinkFlair() -> UserProfileLinkFlair {
         if let userServer = user.actorId.host() {
+            if UserProfileLabel.mlemOfficial == "\(userServer)\(user.actorId.path())" {
```

Comment:
Are we showing instance name for local users?

Reply:
it's an optional toggle